google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -252,6 +252,7 @@ public StorageObject create(StorageObject storageObject, final InputStream conte
           .setMaxResults(Option.MAX_RESULTS.getLong(options))
           .setPageToken(Option.PAGE_TOKEN.getString(options))
           .setFields(Option.FIELDS.getString(options))
+          .setUserProject(Option.FIELDS.getString(options))
```

Comment:
Should this use called `Option.FIELDS` (it is suspicious, because it looks like a copied value from the previous line)

Reply:
Wow that was a glaring error! Thanks for catching that! The unit test is updated too.